www/_eleventy/main-sidebar.njk

```diff
@@ -0,0 +1,323 @@
+<nav class="toc">
```

Comment:
@melissamcewen wanted your thoughts on this: Since we're moving to a consistent, hard-coded sidebar across the site, this is just a temporary fix for the "plugins" page so that it doesn't thrash our current "one pager" sidebar. It's just a copy paste of the existing one pager.